docs/harvesting.rst

```diff
@@ -34,6 +34,11 @@ This document specifies how to harvest records into your system.
 
 If you are going to run harvesting workflows which needs prediction models such as the CORE guessing, keyword extraction, and plot extraction you may need to install some extra packages.
 
+.. warning::
+
+    These services are not Dockerized, so you will have to do that yourself if the need arises.
+    Below instructions are only applicable if you're running inspire locally without Docker.
+
```

Comment:
What services? Beard and Magpie? It's not clear to me from this phrase :/, can you clarify?

(everything else is, scrapyd/web/worker...)